pkg/controller/storageversionmigrator/resourceversion.go

```diff
@@ -284,3 +293,36 @@ func (rv *ResourceVersionController) isResourceNamespaceScoped(gvr schema.GroupV
 
 	return false, fmt.Errorf("resource %q not found", gvr.String())
 }
+
+func (rv *ResourceVersionController) isResourceMigratable(gvr schema.GroupVersionResource) (bool, error) {
```

Comment:
+godoc here?  The key bits to doc are what verbs and required and what conditions result in errors vs (false, nil).